tests/esm-path-mapping/tsconfig-base.json

```diff
@@ -0,0 +1,12 @@
+{
+  "include": ["**/*.ts", "**/*.mts", "**/*.tsx", "**/*.js", "**/*.jsx"],
+  "compilerOptions": {
+    "module": "ESNext",
+    "allowJs": true,
+    "jsx": "react",
+    "moduleResolution": "node"
+  },
+  "ts-node": {
+    "experimentalPathMapping": "both"
```

Comment:
Path mapping is opt-in, so I added this to your tests to enable the path-mapper.

Reply:
Nevermind, I was wrong.  I forgot that path mapping is enabled by default for ESM, but not for CJS.

Reply:
Relevant when I create the CJS tests though, right?